BLE.py

```diff
@@ -0,0 +1,32 @@
+#If not previously paired with an Android device it must be using bluetooth packages
+#sudo bluetoothctl
+#power on
+#discoverable on
+#pairable on
+#pair <MAC_ADDRESS> - pi must respond yes
+
+import bluetooth
+
+class socket:
+    def __init__(socket):
+        socket.server = bluetooth.BluetoothSocket( bluetooth.RFCOMM )
+        port = 1
+        socket.server.bind(("",port))
+
+    def advertise(socket):
+        socket.server.listen(1)
+
+    def connect(socket):
+        socket.client, address = socket.server.accept()
+        print ("Accepted connection from ",address)
```

Comment:
`print` automatically adds a space in between items, so you don't need the space after "from"